samples/browser/shared_subscription/tsconfig.json

```diff
@@ -0,0 +1,64 @@
+{
+    "compilerOptions": {
+        /* Basic Options */
+        "target": "es6", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+        "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+        // "lib": [],                             /* Specify library files to be included in the compilation. */
```

Comment:
Should all these commented options be deleted or are we leaving them here intentionally?

Reply:
I think they were generated at one point. I'll remove it for this PR and when I clean up the other JS samples, I'll remove the commented options accordingly as well.